src/api/forms/templates.js

```diff
@@ -22,13 +26,12 @@ export function empty() {
             schema: {}
           }
         ],
-        next: [{ path: '/summary' }]
+        next: [{ path: ControllerPath.Summary }]
       },
       {
         title: 'Summary',
-        path: '/summary',
-        controller: 'SummaryPageController',
-        components: []
```

Comment:
We no longer support `components: []` in summary pages

See similar changes in **forms-runner** https://github.com/DEFRA/forms-runner/pull/361/commits/6b9653776b37867fbde2839e759bfb936f48aae4